patches/cover_patch.diff

```diff
@@ -6,28 +6,22 @@ Since Encore's code generation adds some extra files, this doesn't quite work.
 This patch fixes this by skipping over these files, and reading the source file
 using the overlay file system instead of directly from disk.
 
-diff --git a/src/cmd/go/internal/test/test.go b/src/cmd/go/internal/test/test.go
-index 59ea1ef544..d603a29939 100644
---- a/src/cmd/go/internal/test/test.go
-+++ b/src/cmd/go/internal/test/test.go
-@@ -1056,7 +1056,7 @@ func declareCoverVars(p *load.Package, files ...string) map[string]*load.CoverVa
- 	sum := sha256.Sum256([]byte(p.ImportPath))
- 	h := fmt.Sprintf("%x", sum[:6])
- 	for _, file := range files {
--		if isTestFile(file) {
-+		if isTestFile(file) || strings.Contains(file, "encore_internal__") {
- 			continue
- 		}
- 		// For a package that is "local" (imported via ./ import or command line, outside GOPATH),
```

Comment:
I've removed this patch because the `declareCoverVars` has been removed in Go 1.20. I'll try and figure out whether we need to make other changes to fix coverage.

Reply:
Sounds good